src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -26,6 +26,7 @@
 [assembly: InternalsVisibleTo("OrleansProviders")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansServiceBus")]
```

Comment:
or I can just use .net Timer class.... 
What's the history of SafeTimer? Its description makes me think it is better than Timer....